src/Movie/Movie.js

```diff
@@ -20,46 +20,16 @@ class Movie extends Component {
             tagline: '',
             averageRating: ''
         },
-         fetchedMovies: [],
```

Comment:
Removed this and all the code that stores movies because it doesn't work with the force update of the app